package.json

```diff
@@ -28,9 +28,9 @@
   "author": "collaboration Factory AG",
   "license": "ISC",
   "dependencies": {
-    "@openapitools/openapi-generator-cli": "2.1.26",
+    "@openapitools/openapi-generator-cli": "^2.4.26",
```

Comment:
I`m not sure, maybe we should switch to pinpoint versions? (After all these hijacked npm packages)